docker/Dockerfile.wifi-connect

```diff
@@ -0,0 +1,43 @@
+# @TODO: Make use of build args or `sed` to customize the
+#     base image to be used.
+FROM balenalib/aarch64-python as builder
```

Comment:
This is not a multi-stage build. Since it's not, please don't mix base images as that will add a lot of additional storage on the host.